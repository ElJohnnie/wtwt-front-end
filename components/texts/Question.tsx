export interface QuestionProps {
  text: string;
}

export default function Question(props: QuestionProps) {
  const { text } = props;

  return (
    <div
      className={`py-2 px-12 rounded-full font-bold text-pink-600 text-center text-2xl`}
    >
      {text}
    </div>
  );
}