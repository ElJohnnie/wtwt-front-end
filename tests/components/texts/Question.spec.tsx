import React from 'react';
import Question from '../../../components/texts/question.component';
import '@testing-library/jest-dom';
import { renderWithOutContextProvider } from '../../../utils/tests/helpers';

describe('Description', () => {
  it('renderizar o component Question', () => {
    const { container, getByText } = renderWithOutContextProvider(
      <Question text="batatinha?" />,
    );

    const receipt = getByText('batatinha?');
    expect(receipt).toBeInTheDocument();
    expect(container).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
