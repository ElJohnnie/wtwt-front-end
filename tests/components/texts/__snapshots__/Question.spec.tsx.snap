// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renderizar o component Question 1`] = `
<div>
  <div
    class="py-2 px-2 md:px-12 rounded-full text-center"
  >
    <h2
      class="MuiTypography-root MuiTypography-h2 css-1bkebpk-MuiTypography-root"
    >
      batatinha?
    </h2>
  </div>
</div>
`;
